src/azure-cli/HISTORY.rst

```diff
@@ -135,6 +135,7 @@ Release History
 * Add a new argument `--expose-token` for `az acr login`
 * Fix the incorrect output of `az acr task identity show -n Name -r Registry -o table`
 * az acr login: Throw a CLIError if there are errors returned by docker command
+* Add a new command group 'az acr task agentpool create/update/show/list/delete' for taskrun management
```

Comment:
history.rst file is automatically generated from PR title now, no need to manual update

Reply:
thanks

Reply:
thanks